packages/manager/apps/hycu/src/index.tsx

```diff
@@ -28,7 +28,7 @@ const init = async (appName: string) => {
     context,
     reloadOnLocaleChange: true,
     defaultNS: appName,
-    ns: ['listing', 'dashboard', 'onboarding'],
+    ns: ['listing', 'dashboard', 'onboarding', 'order'],
```

Comment:
That's the namespaces that are loaded directly when starting the app
You don't have to put all namespaces here